New docs

acmenezes opened the pull request:
First iteration on new README for pod-network-operator.

acmenezes committed:
ADD new README2 file to be renamed when ready

acmenezes committed:
ADD text to README2

acmenezes committed:
New README.md

williamcaban commented on README.md:
```diff
@@ -1,60 +1,69 @@
----
+With those elements being painted on the scenario the need for more dynamic, flexible and intelligent networks capable of making changes or scheduling resources based on telemetry data becomes something real. Being able to create networks on the fly and delete them as a 5G call terminates, slice networks through multiple hops to separate traffic, classify packets and traffic and treat them accord ...
+
+The goal of the pod-network-operator is to experiment, try and test innovative ways to do networking, basically re-inventing Kubernetes networking, if we can put like that, for the new tech community writing this chapter in history.
```
I won't call it re-inventing because it is not replacing anything, it is providing alternate ways to do networking while maintaining compatibility with the the K8s constructs and network requirements (e.g. no NAT for pod-to-pod communication, etc)

acmenezes replied:
Agreed. Working on better way to put this. Thanks!

acmenezes committed:
UPDATE docs based on comment

https://github.com/fennec-project/pod-network-operator/pull/2#discussion_r684493812

williamcaban commented:
/lgtm
